docs/smart_contract.md

```diff
@@ -0,0 +1,104 @@
+TODO: 
+1. Explain how to sync up the smart contract with the DKG
+2. Explain how to sync up the smart contract with the Shuffle
+3. Explain how security impacts the smart contract (at the decrypt stage)
+
+## Create an election
+This transaction requires the following 3 parameters:
+1. *title* of the election
+2. *admin* ID of the creator of the election
+3. *format* of the election
+
+Key / Value pairs sent in the transaction in order to create an election:
+| | |
+|-|-|
+|"go.dedis.ch/dela.ContractArg"|[]byte(evoting.ContractName)|
+|evoting.CreateElectionArg|createElectionBuf|
+|evoting.CmdArg|[]byte(evoting.CmdCreateElection|
+
+where:
+    
+    evoting.ContractName = "go.dedis.ch/dela.Evoting"
```

Comment:
We use the \`\`\` syntax for code. You can then specify the language: https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting